app/(auth)/(tabs)/transactions.tsx

```diff
@@ -1,102 +1,308 @@
+        <ThemedView style={[styles.row, styles.shadowProp, {backgroundColor: theme.colors.card}]}>
+        <ThemedText style={styles.subHeading2Text}>    Total # Issued  </ThemedText>
+            <ThemedView style={[styles.transactionContainer, {backgroundColor: theme.colors.card}]}>
+              <Image
+                source={require('@/assets/images/1Point_Logo.png')}
+                style={styles.pointAmounts}
+              />
+              <ThemedText style={[styles.transactionPreviewText, {color: theme.colors.text}]}>178</ThemedText>
+            </ThemedView>
+            <ThemedView style={styles.backLabelContainerUp}>
+                  <ThemedText style={styles.backLabel}>+2% - Last Month</ThemedText>
+                </ThemedView>
+          </ThemedView>
+
 
-export default function TabTwoScreen() {
+        </ThemedView>
+          
+  
+        </ThemedView>
+      );
+    };
+
+    const mapTransactions = () => {
+      return (
+        <ThemedView>
+          {transactionArray.map((transaction) => {
+            return (
+              <TransactionRow transactionAmount={transaction.transactionAmount} transactionLocation={transaction.transactionLocation} transactionDate={transaction.transactionDate} transactionCustomerId={transaction.transactionCustomerId} transactionStatus={transaction.transactionStatus}/>
+            );
+          })}
+        </ThemedView>
+      );
+    }
+
+  // Render the transactions section
+  const renderTransactions = () => {
+    return (
+      <ThemedView style={{paddingBottom: 120, backgroundColor: theme.colors.background}}>
+       {mapTransactions()}
+      </ThemedView>
+    );
+  };
+
+  // Render the home screen 
   return (
-    <ParallaxScrollView
-      headerBackgroundColor={{ light: '#D0D0D0', dark: '#353636' }}
-      headerImage={<Ionicons size={310} name="code-slash" style={styles.headerImage} />}>
-      <ThemedView style={styles.titleContainer}>
-        <ThemedText type="title">Transaction</ThemedText>
+    <SafeAreaView style={[styles.main, {backgroundColor: theme.colors.background}]}>
+      <ScrollView style={[styles.mainContainer, {backgroundColor: theme.colors.background}]}>
+
+        <ThemedView style={[styles.headerContainer, {backgroundColor: theme.colors.background}]}>
+          <Image
+            source={require('@/assets/images/1Point_Logo.png')}
+            style={styles.headerImage}
+          />
+          <ThemedView style={[styles.headerText, {backgroundColor: theme.colors.background}]}>
+            <ThemedText style={[styles.welcomeText]}>TRANSACTIONS</ThemedText>
+          </ThemedView>
+        </ThemedView>
+        <ThemedText style={styles.subHeadingText}>TRANSACTION PREVIEW</ThemedText>
+        <ThemedView>
+          {renderTransactionPreview()}
+          </ThemedView>
+
+        <ThemedText style={styles.subHeadingText}>TRANSACTION LIST</ThemedText>
+
+
+      <ThemedView>
```

Comment:
this can be our scroll view

Reply:
Trying a few styles out. I think that the transaction preview should be included in the scroll view and we have a common page header. Committing an update real quick.